Documentation/sourcebuild-in-repos/new-repo.md

```diff
@@ -1,11 +1,10 @@
 # Adding a new repository to source build
 
-This document describes the overall process of onboarding new repos onto
-arcade powered source build (ArPow).
+This document describes the overall process of onboarding new repos onto source build.
```

Comment:
There should be another step for updating the `CODEOWNERS` file with this, if the repo has that file:

```
/eng/SourceBuild* @dotnet/source-build-internal
```

Reply:
Added a CODEOWNERS section for this.